docs/identity/hybrid/connect/how-to-connect-install-prerequisites.md

```diff
@@ -105,6 +105,10 @@ We recommend that you harden your Microsoft Entra Connect server to decrease the
   * You can have only one sync engine per SQL instance. Sharing a SQL instance with MIM Sync, DirSync, or Azure AD Sync *isn't supported*.
   * Maintain ODBC Driver for SQL Server version 17 and OLE DB Driver for SQL Server version 18 that are bundled with Microsoft Entra Connect. Upgrading ODBC/OLE DB drivers’s major or minor version aren't supported. Microsoft Entra Connect product group team will include new ODBC/OLE DB drivers as these become available and have a requirement to be updated.
 
+> [!NOTE]
+> If your are installing SQL on the same server as Microsoft Entra Connect, we recommend to configure SQL to limit the maximum memory that it can use from the system.
```

Comment:
```suggestion
> If you're installing SQL on the same server as Microsoft Entra Connect, we recommend to configure SQL to limit the maximum memory that it can use from the system.
```